Lib/glyphsLib/builder/tokens.py

```diff
@@ -2,6 +2,13 @@
 from collections import OrderedDict
 
 
+def _like(got, expected):
+    expected = expected.replace("?", ".")
+    expected = expected.replace("*", ".*")
+    # Technically we should be a bit stricter than this
```

Comment:
How so?

Reply:
Because if the RHS of the predicate (`expected`) might contain other characters which are meaningful within a regular expression match. e.g. `$[name like "haha this is an invalid re ("]` will cause it to explode.